CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v10.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v10.0.0...HEAD)] - 2020-xx-xx
 
-- Write your awesome change here (GH-xxxx)
+- Add `create_eks = false` example. (@dpiddockcmp)
```

Comment:
please remove your change from changelog.

Reply:
Done. Thanks for the cool new auto changelog